types/node/test/async_hooks.ts

```diff
@@ -98,3 +99,125 @@ import {
+    // $ExpectType 8
+    asyncWrapProviders.FSREQCALLBACK;
+    // $ExpectType 9
+    asyncWrapProviders.FSREQPROMISE;
+    // $ExpectType 10
+    asyncWrapProviders.GETADDRINFOREQWRAP;
+    // $ExpectType 11
+    asyncWrapProviders.GETNAMEINFOREQWRAP;
+    // $ExpectType 12
+    asyncWrapProviders.HEAPSNAPSHOT;
+    // $ExpectType 13
+    asyncWrapProviders.HTTP2SESSION;
+    // $ExpectType 14
+    asyncWrapProviders.HTTP2STREAM;
+    // $ExpectType 15
+    asyncWrapProviders.HTTP2PING;
+    // $ExpectType 16
+    asyncWrapProviders.HTTP2SETTINGS;
+    // $ExpectType 17
+    asyncWrapProviders.HTTPINCOMINGMESSAGE;
+    // $ExpectType 18
+    asyncWrapProviders.HTTPCLIENTREQUEST;
+    // $ExpectType 19
+    asyncWrapProviders.JSSTREAM;
+    // $ExpectType 20
+    asyncWrapProviders.JSUDPWRAP;
+    // $ExpectType 21
+    asyncWrapProviders.MESSAGEPORT;
+    // $ExpectType 22
+    asyncWrapProviders.PIPECONNECTWRAP;
+    // $ExpectType 23
+    asyncWrapProviders.PIPESERVERWRAP;
+    // $ExpectType 24
+    asyncWrapProviders.PIPEWRAP;
+    // $ExpectType 25
+    asyncWrapProviders.PROCESSWRAP;
+    // $ExpectType 26
+    asyncWrapProviders.PROMISE;
+    // $ExpectType 27
+    asyncWrapProviders.QUERYWRAP;
+    // $ExpectType 28
+    asyncWrapProviders.SHUTDOWNWRAP;
+    // $ExpectType 29
+    asyncWrapProviders.SIGNALWRAP;
+    // $ExpectType 30
+    asyncWrapProviders.STATWATCHER;
+    // $ExpectType 31
+    asyncWrapProviders.STREAMPIPE;
+    // $ExpectType 32
+    asyncWrapProviders.TCPCONNECTWRAP;
+    // $ExpectType 33
+    asyncWrapProviders.TCPSERVERWRAP;
+    // $ExpectType 34
+    asyncWrapProviders.TCPWRAP;
+    // $ExpectType 35
+    asyncWrapProviders.TTYWRAP;
+    // $ExpectType 36
+    asyncWrapProviders.UDPSENDWRAP;
+    // $ExpectType 37
+    asyncWrapProviders.UDPWRAP;
+    // $ExpectType 38
+    asyncWrapProviders.SIGINTWATCHDOG;
+    // $ExpectType 39
+    asyncWrapProviders.WORKER;
+    // $ExpectType 40
+    asyncWrapProviders.WORKERHEAPSNAPSHOT;
+    // $ExpectType 41
+    asyncWrapProviders.WRITEWRAP;
+    // $ExpectType 42
+    asyncWrapProviders.ZLIB;
+    // $ExpectType 43
+    asyncWrapProviders.CHECKPRIMEREQUEST;
+    // $ExpectType 44
+    asyncWrapProviders.PBKDF2REQUEST;
+    // $ExpectType 45
+    asyncWrapProviders.KEYPAIRGENREQUEST;
+    // $ExpectType 46
+    asyncWrapProviders.KEYGENREQUEST;
+    // $ExpectType 47
+    asyncWrapProviders.KEYEXPORTREQUEST;
+    // $ExpectType 48
+    asyncWrapProviders.CIPHERREQUEST;
+    // $ExpectType 49
+    asyncWrapProviders.DERIVEBITSREQUEST;
+    // $ExpectType 50
+    asyncWrapProviders.HASHREQUEST;
+    // $ExpectType 51
+    asyncWrapProviders.RANDOMBYTESREQUEST;
+    // $ExpectType 52
+    asyncWrapProviders.RANDOMPRIMEREQUEST;
+    // $ExpectType 53
+    asyncWrapProviders.SCRYPTREQUEST;
+    // $ExpectType 54
+    asyncWrapProviders.SIGNREQUEST;
+    // $ExpectType 55
+    asyncWrapProviders.TLSWRAP;
+    // $ExpectType 56
+    asyncWrapProviders.VERIFYREQUEST;
```

Comment:
Individual unwrapped tests are probably unnecessary – it'd be adding nearly 200 additional eslint checks to each CI run. A single example of accessing a property key is probably fine.

Reply:
Done.